auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -252,6 +265,25 @@ else if (type.equals(SIZEF))
     return block.build();
   }
 
+  public static CodeBlock writeValueWithTypeAdapter(FieldSpec adapter, AutoValueParcelExtension.Property property, ParameterSpec out) {
+    CodeBlock.Builder block = CodeBlock.builder();
+
+    if (property.nullable()) {
+      block.beginControlFlow("if ($N() == null)", property.methodName);
+      block.addStatement("$N.writeInt(1)", out);
```

Comment:
I think this convention comes from `Parcel` itself. They use 1 to indicate null.